### PR TITLE
Styled the user profile page and components

### PR DESCRIPTION
**Pull request by aOrro**
Worked on the styling of the User page and related components.
Added some more images to the assets/images folder.

### CODE REVIEW

File: src/components/Collection/index.js [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
-      <span>{data.total_photos} photos</span>
-    </div>
+      <CollectionInfo>
+        <h3>{data.title}</h3>
+        <span>{data.total_photos} photos</span>
+        <Labels>
+          <Label>{initialToUppercase(data.tags[0].title)}</Label>
+          <Label>{initialToUppercase(data.tags[1].title)}</Label>
+          <Label>{initialToUppercase(data.tags[2].title)}</Label>
+        </Labels>
```
Comment (yoloOnTheBattlefield) [inline src/components/Collection/index.js:21]: .map

File: src/pages/User/index.js [hunk 2]
```diff
@@ -4,14 +4,20 @@ import { UserHeader } from '../../components/UserHeader';
...
   state = {
-    inputValue: '',
     userInfo: null,
     userPhotos: [],
     userCollections: [],
     userStats: null,
+    showPhotos: true,
+    showCollections: false,
+    showStats: false,
   };
```
Comment (yoloOnTheBattlefield) [inline src/pages/User/index.js:25]: you don't have a loading flag or something? did you forget about the setup?
Watch the video again: https://floreacristian.teachable.com/courses/from-beginner-to-professional-developer1/lectures/31280909?preview=logged_in

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/Collection/index.js (hunk 1 above)
Comment (aOrro) [reply src/components/Collection/index.js:21]: why map? I just want the 3 first items from the array not all of them...

File: src/pages/User/index.js (hunk 2 above)
Comment (aOrro) [reply src/pages/User/index.js:25]: Yeah I should definitely add that, thanks!

File: src/components/PhotoCard/index.js [hunk 3]
```diff
@@ -22,11 +22,11 @@ export const PhotoCard = props => {
       <Photo src={props.src} alt={props.alt} />
       <PhotoCardFooter>
         <Likes>
-          <img src={heartLogo} alt='heart' style={{ marginRight: 5 }} />
+          <img src={heartIcon} alt='heart' style={{ marginRight: 5 }} />
```
Comment (vladciuca) [inline src/components/PhotoCard/index.js:25]: styleszzz in  JSX

**Pull request merged**